### DISCUSSION TITLE
search channel option and filter suggestion

### DISCUSSION THREAD

**Discussion by helloflatearth** [InfinityLoop1308/PipePipe#134] [Ideas]
I think there should be a search channel feature.

Also, I think it would be cool in general search if you could specify view counts when filtering by a range. 

For instance, search for videos with more than 2,500 views, or even use a range, like between 2,500 and 5,000 views.

This would give the user more flexibility to find amateur content and videos that usually get overlooked by the YouTube algorithm, while not pulling up a million garbage videos with basically 0 views.

**Comment by InfinityLoop1308** [comment]
It is impossible, unfortunately.
